pkg/testutils/lint/lint_test.go

```diff
@@ -546,6 +546,38 @@ func TestLint(t *testing.T) {
 		}
 	})
 
+	t.Run("TestAmbientContextConstruction", func(t *testing.T) {
+		t.Parallel()
+		cmd, stderr, filter, err := dirCmd(
+			pkgDir,
+			"git",
+			"grep",
+			"-nE",
+			`[^[:alnum:]]log\.AmbientContext\{`,
+			"--",
+			"*.go",
+		)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		if err := cmd.Start(); err != nil {
+			t.Fatal(err)
+		}
+
+		if err := stream.ForEach(filter, func(s string) {
+			t.Errorf("\n%s <- forbidden; reuse an existing AC when possible, or use one of the constructor functions instead: log.MakeServerAC(), log.MakeClientAC() or testutils.MakeAC()", s)
```

Comment:
please make sure there's a pragma that can be used to inhibit the linter at a call site.